app/services/facemasks.ts

```diff
@@ -213,9 +213,11 @@ export class FacemasksService extends PersistentStatefulService<IFacemasksServic
   }
 
   trigger(mask: string, message: string, name: string) {
+    const clean = this.profanitize(message, name);
```

Comment:
I know that `profanitize` is a portmanteau of `profanity` and `sanitize`, but it sounds like it's going to sprinkle some profanity into whatever string you pass it.  We should write that function some day.  `function realProfanitize(msg)`.  😄

Reply:
Yeah it does sound like some kind of markov profanity generator, maybe call it `deProfanitize`